doc/contributing.rst

```diff
@@ -130,20 +148,25 @@ dependencies as described in our :doc:`parallel backend guide <parallel>`.
 Updating documentation
 ======================
 
-Update ``doc/api.rst`` and ``doc/whats_new.rst`` as appropriate.
+When you update the documentation, it is recommended to build it locally to
+check whether the documentation renders correctly in HTML.
+
+Certain documentation files require frequent updates. These are::
+
+    * ``doc/api.rst`` if you added a new function.
+    * ``doc/whats_new.rst`` to document the fix or change so you can be
+      credited on the next release.
+
+Please update these documents at the end to avoid rebase conflicts with other
+pull requests.
 
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The documentation can be built using sphinx. For that, please additionally
 install the following::
 
-    $ pip install matplotlib sphinx numpydoc sphinx-gallery sphinx_bootstrap_theme pillow mpi4py joblib psutil nbsphinx
-
-If you are using a newer version of pip, you may be prompted to use the flag
-``--use-feature=2020-resolver``. If this happens, please add it as recommended::
-
-    $ pip install --use-feature=2020-resolver matplotlib sphinx numpydoc sphinx-gallery sphinx_bootstrap_theme pillow mpi4py joblib psutil nbsphinx
+    $ pip install matplotlib sphinx numpydoc sphinx-gallery sphinx_bootstrap_theme pillow joblib psutil nbsphinx
```

Comment:
mpi4py doesn't install like this ... you'll have to follow the guide for parallel stuff

Reply:
How so? I'm pretty sure this is how I installed mpi4py.... Either that or it was through conda.

Reply:
Also, it might be important to indicate to new contributors that successful completion of all our tests require MPI.

Reply:
> I'm pretty sure this is how I installed mpi4py.... Either that or it was through conda.

There are subtle differences depending on the OS and the compilers available. This command doesn't work "out of the box"

> Also, it might be important to indicate to new contributors that successful completion of all our tests require MPI.

Agreed, I will highlight that!